.gitignore

```diff
@@ -19,3 +19,5 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+/.idea/
```

Comment:
This line should be in [your global .gitignore](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c)